src/index.html

```diff
@@ -30,6 +30,54 @@ <h3>Login</h3>
         <h3>Your Bookings</h3>
       </section>
     </section>
+    <section tabindex=0 class="customer-search-dashboard hidden">
+      <section class="select-a-date">
+        <p>What date would you like to stay at the Overlook Hotel?</p></br>
+        <input type="date" name="date" class="room-search-date"></input></br>
+        <button class="search-room-button">Search Rooms</button>
+      </section>
+      <section class="return-customer-dashboard">
+        <p>Return to your page</p>
+        <button class="return-customer-page-button">Return</button>
+      </section>
+      <section tabindex= 0 class="room-filter-buttons">
+        <button class="all-rooms-button" aria-label="Select all room types" alt="select all room types button">All Rooms</button>
```

Comment:
Really nice use of aria-label!! I've added a note in my journal to go back and reformat my buttons to make them accessible like this. Accessibility continues to be a strong focus of your code- well done!!